libethash-cl/CLMiner.cpp

```diff
@@ -266,6 +266,17 @@ CLMiner::~CLMiner()
     kick_miner();
 }
 
+typedef struct {
```

Comment:
In C++ you don't need to use `typedef` for `struct`.

Reply:
?